tests/ops/qubit/test_sparse.py

```diff
@@ -120,7 +120,7 @@ def circuit(param):
             match="SparseHamiltonian observable must be"
             " used with the parameter-shift differentiation method",
         ):
-            qml.grad(circuit, argnum=0)([0.5])
+            qml.grad(circuit, argnum=0)(0.5)
```

Comment:
Added as part of the recent update to autoray, this should be removed once master is updated with the latest fix for it. Until then, it is kept here so that CI can pass.